core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/ModuleConfigUpdateExecutor.java

```diff
@@ -42,22 +45,50 @@
 import org.sonarsource.sonarlint.core.util.StringUtils;
 import org.sonarsource.sonarlint.core.util.VersionUtils;
 
+import static org.sonarsource.sonarlint.core.container.connected.update.IssueUtils.groupByFileKey;
+
 public class ModuleConfigUpdateExecutor {
 
   private final StorageManager storageManager;
   private final SonarLintWsClient wsClient;
+  private final IssueDownloader issueDownloader;
+  private final IssueStoreFactory issueStoreFactory;
   private final TempFolder tempFolder;
 
   public ModuleConfigUpdateExecutor(StorageManager storageManager, SonarLintWsClient wsClient, TempFolder tempFolder) {
+    // TODO replace InMemoryIssueStore with persistent (filesystem-based) implementation (in progress)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)